templates/puppetserver_metrics.epp

```diff
@@ -18,14 +18,23 @@ HOSTS = [
 
 filename = Time.now.utc.strftime('%Y%m%dT%H%M%SZ') + '.json'
 
+def get_status_endpoint(host)
+  uri = URI.parse("https://#{host}:8140/status/v1/services?level=debug")
+  metrics = open(uri, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE}).read
+  data = JSON.parse(metrics)
+  data = data.merge({"error" => "", "error_count" => 0})
+  rescue Exception => e
```

Comment:
Looks like the `begin; ...; rescue; ...; end` structure got broken here when the method was split out. Needs to be fixed